### PR TITLE
feat: web docs

### PR DESCRIPTION
**Pull request by Pajk**

### CODE REVIEW

**Comment by Pajk** [comment]
I moved here all the pages from the current web sdk documentation.

The only thing missing are titles and subtitles on the page and we should create a proper getting started page.

File: src/web/index.mdx [hunk 1]
```diff
@@ -25,10 +25,10 @@ Copy the tracking code from your project settings and include it on your web. Th
 
 Check the `GDPR compliant by design` section and adjust the settings if needed. If you plan to identify visitors from EU you have to sign our DPA, read more on that in the GDPR section.
 
-
 ## Further Steps
 
 <TextBlock>{`
+* Check our <Link target="_blank" to="https://help.smartlook.com">Knowledge base</Link> for more general information, integrations and FAQ.
```
Comment (mod3x) [inline src/web/index.mdx]: I think we can remove **TextBlock** here, also change **Link** with html **a** tag as It's not an internal app link

Comment (Pajk) [reply src/web/index.mdx]: done

Comment (mod3x) [reply src/web/index.mdx]: cool

**Pull request merged**